src/Common/src/CoreLib/System/Buffers/StandardFormat.cs

```diff
@@ -71,8 +71,30 @@ public StandardFormat(char symbol, byte precision = NoPrecision)
         /// </summary>
         public static StandardFormat Parse(ReadOnlySpan<char> format)
         {
+            ParseHelper(format, out StandardFormat standardFormat, true);
+
+            return standardFormat;
+        }
+
+        /// <summary>
+        /// Converts a classic .NET format string into a StandardFormat
+        /// </summary>
+        [System.Obsolete]public static StandardFormat Parse(string format) => format == null ? default : Parse(format.AsSpan());
```

Comment:
Why is this being marked as Obsolete?  I don't see that as having been part of the approved changes, and we generally do not do that.

Reply:
I see. Will change it back.

As I mentioned [here](https://github.com/dotnet/corefx/issues/32950#issuecomment-441552233), I also think now, that the string overloads could still be quite important for some use cases. What is your idea?